From 80f20324c09928dc20a2c19f02e0dc3abe10dc00 Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Mon, 8 Sep 2014 15:07:50 -0400
Subject: [ethernet] ixgbevf: Convert uses of __constant_<foo> to <foo>

Message-id: <1410188883-6595-36-git-send-email-jogreene@redhat.com>
Patchwork-id: 92388
O-Subject: [RHEL7.1 PATCH 35/48] ixgbevf: Convert uses of __constant_<foo> to <foo>
Bugzilla: 1091124
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: 1091124
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7919168
Tested: In beaker. See BZ for details

The use of __constant_<foo> has been unnecessary for quite awhile now.

Make these uses consistent with the rest of the kernel.

Signed-off-by: Joe Perches <joe@perches.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
(cherry picked from commit 0933ce4a9d68ae35dccfa592fa361c149df5d1af)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c b/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c
index 5c41dc3..a4c94e3 100644
--- a/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c
+++ b/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c
@@ -2857,12 +2857,12 @@ static void ixgbevf_tx_csum(struct ixgbevf_ring *tx_ring,
  if (skb->ip_summed == CHECKSUM_PARTIAL) {
   u8 l4_hdr = 0;
   switch (skb->protocol) {
-  case __constant_htons(ETH_P_IP):
+  case htons(ETH_P_IP):
    vlan_macip_lens |= skb_network_header_len(skb);
    type_tucmd |= IXGBE_ADVTXD_TUCMD_IPV4;
    l4_hdr = ip_hdr(skb)->protocol;
    break;
-  case __constant_htons(ETH_P_IPV6):
+  case htons(ETH_P_IPV6):
    vlan_macip_lens |= skb_network_header_len(skb);
    l4_hdr = ipv6_hdr(skb)->nexthdr;
    break;
-- 
1.7.1